$OpenBSD: patch-lib_Bio_ASN1_EntrezGene_pm,v 1.1 2019/08/09 16:03:10 cwen Exp $

Fix unescaped left braces so it does not generate warnings with Perl 5.28

Index: lib/Bio/ASN1/EntrezGene.pm
--- lib/Bio/ASN1/EntrezGene.pm.orig
+++ lib/Bio/ASN1/EntrezGene.pm
@@ -270,7 +270,7 @@ sub next_seq
   {
     chomp;
     next unless /\S/;
-    my $tmp = (/^\s*Entrezgene ::= ({.*)/si)? $1 : "{" . $_; # get rid of the 'Entrezgene ::= ' at the beginning of Entrez Gene record
+    my $tmp = (/^\s*Entrezgene ::= (\{.*)/si)? $1 : "{" . $_; # get rid of the 'Entrezgene ::= ' at the beginning of Entrez Gene record
     return $self->parse($tmp, $compact, 1); # 1 species no resetting line number
   }
 }
@@ -324,7 +324,7 @@ sub _parse
         elsif($data->{$id}) { $data->{$id} = [$data->{$id}, $value] } # hash value has a second terminal value now!
         else { $data->{$id} = $value } # the first terminal value
       }
-      elsif($self->{input} =~ /\G{/cg)
+      elsif($self->{input} =~ /\G\{/cg)
       {
         $self->{depth}++;
         push(@{$data->{$id}}, $self->_parse());
